tiered-storage/jcloud/src/main/java/org/apache/bookkeeper/mledger/offload/jcloud/BackedInputStream.java

```diff
@@ -18,9 +18,12 @@
  */
 package org.apache.bookkeeper.mledger.offload.jcloud;
 
-import java.io.InputStream;
 import java.io.IOException;
+import java.io.InputStream;
 
+/**
```

Comment:
Unrelated change.

Reply:
This change, along with several others you have flagged as "unrelated" are actually changes made to conform with the check style rules defined in the pulsar/buildtools/src/main/resources/pulsar/checkstyle.xml file.  

Before I commit my changes I run "mvn -Pcontrib-check clean install" , which flags any check style violations.